Project_source/app/src/main/java/android/clicker/school_live_simulator/NullGuitarState.kt

```diff
@@ -0,0 +1,9 @@
+package android.clicker.school_live_simulator
+
+open class NullGuitarState : GuitarState() {
+    override val price: Int = 0
+
+    override fun changeState(bag: Player.Bag) {
+        bag.setGuitar(UssrGuitarState())
```

Comment:
bag.guitar is public there will be no method setGuitar. There is a note in UML about that
![image](https://user-images.githubusercontent.com/58232044/128497539-c3025c89-e163-4d48-af13-0b20750b0beb.png)

Reply:
So, why in UML we have this instead public field with no getters and setters? After all, will the field be public, and should I fix it to assignment operator, as it was before?
![image](https://user-images.githubusercontent.com/72977819/128513723-432e1679-ee43-45d9-b5b9-8b0785965cd7.png)

Reply:
> So, why in UML we have this instead public field with no getters and setters? After all, will the field be public, and should I fix it to assignment operator, as it was before?
> ![image](https://user-images.githubusercontent.com/72977819/128513723-432e1679-ee43-45d9-b5b9-8b0785965cd7.png)

Composition on your screen is the same as on my screen and there is information about real realisation in Kotlin. It was in our file when these setters and getters were added, so I don't understand why do you have a question about that? We will realise Kotlin's set() for changing Bag's items and Courses' courses but all items in bag are public. @Ferrociraptus

Reply:
Fixed